android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/ValidatePinWithPolicyUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.onegini.mobile.sdk.reactnative.clean.use_cases
+
+import com.facebook.react.bridge.Promise
+import com.onegini.mobile.sdk.reactnative.OneginiSDK
+import com.onegini.mobile.sdk.android.handlers.OneginiPinValidationHandler
+import com.onegini.mobile.sdk.android.handlers.error.OneginiPinValidationError
+import com.onegini.mobile.sdk.reactnative.exception.OneginiWrapperErrors
+
+
+class ValidatePinWithPolicyUseCase(private val oneginiSDK: OneginiSDK) {
+  operator fun invoke(pin: String?, promise: Promise) {
+    if (pin == null) {
+      promise.reject(OneginiWrapperErrors.PARAMETERS_NOT_CORRECT.code, "Expected parameter 'pin' to be String but was NULL")
+      return
+    }
```

Comment:
null check is not needed here. When null pin is provided SDK will return own error.